app/jobs/base_job.rb

```diff
@@ -10,6 +10,15 @@ class BaseJob < ApplicationJob
   include NotifierConfig
   include MaintenanceTaskInstrumentation
 
+  # Priority constants for job scheduling
+  # Lower numbers = higher priority (processed first)
```

Comment:
Let's revisit these names, but I like the idea.

Reply:
That's probably fine, and maybe it doesn't matter a ton, but if I ran into `BULK_PROCESSING_PRIORITY` and `LARGE_BATCH_PRIORITY` in the codebase I'd still need to lookup which was higher priority.  (Same for some of the others that are close together.  Maybe we should add the actual number to the end? `BULK_PROCESSING_PRIORITY_9` and `LARGE_BATCH_PRIORITY_10`.  That way we'd know the intention _and_ the actual outcome when using/finding it.